### PR TITLE
[Maintenance] Sync Master back to Dev when code is pushed to master

### PR DESCRIPTION
**Pull request by PromoFaux**
**By submitting this pull request, I confirm the following:** 

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [ ] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
**What does this PR aim to accomplish?:**

Title - it's pointing at the `master` branch on purpose

### CODE REVIEW

**Comment by PromoFaux** [comment]
If it lands here, I'll do PRs for the other repos

**Review by DL6ER — DISMISSED** [review]

File: .github/release.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+changelog:
+  exclude:
+    labels:
+      - internal
+    authors:
+      - dependabot
+      - github-actions
```
Comment (dschaper) [inline .github/release.yml:7]: Is this an OR condition or AND?

Will it only exclude that label when it's from those authors?

Comment (PromoFaux) [reply .github/release.yml:7]: It's OR

Comment (PromoFaux) [reply .github/release.yml:7]: At least, I think it is!

Comment (PromoFaux) [reply .github/release.yml:7]: Yes it's OR. I have seen it ignore dependabot PRs on adminlte repo, and those were not labelled, too.

**Review by dschaper — DISMISSED** [review]

**Review by DL6ER — APPROVED** [review]

**Review by DL6ER — CHANGES_REQUESTED** [review]
The target branch is `devel` on this repo

**Review by DL6ER — APPROVED** [review]

**Review by dschaper — DISMISSED** [review]

**Review by dschaper — APPROVED** [review]

**Pull request merged**